packages/matchbox/src/components/ProgressBar/styles.js

```diff
@@ -0,0 +1,40 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const outerBase = () => `
+  position: relative;
+  background-color: ${tokens.color_gray_300};
+  border-radius: ${tokens.borderRadius_100};
+  width: 100%;
+  margin-bottom: 0.75rem;
+`;
+
+export const innerBase = () => `
+  width: 100px;
+  background-color: ${tokens.color_blue_700};
+  border-radius: ${tokens.borderRadius_100} ${tokens.borderRadius_0} ${tokens.borderRadius_0} ${tokens.borderRadius_100}
```

Comment:
The `100` value for the border radius is very slight - almost unnoticeable in this situation. Might be worth trying a larger value? Totally subjective though.

Reply:
I'm gonna leave as is since it's specified in the design files this way